src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -1,35 +1,79 @@
-import { Link, Skeleton, Typography } from '@mui/material';
+import { Box, Link, Skeleton, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
+import { fetchPerson } from '../../../api/cristinApi';
 import { fetchRegistration } from '../../../api/registrationApi';
-import { Registration, RegistrationStatus } from '../../../types/registration.types';
+import { ProfilePicture } from '../../../components/ProfilePicture';
+import { PublicationNote, Registration, RegistrationStatus } from '../../../types/registration.types';
+import { getIdentifierFromId } from '../../../utils/general-helpers';
+import { getFullCristinName } from '../../../utils/user-helpers';
 import { StyledStatusMessageBox } from '../../messages/components/PublishingRequestMessagesColumn';
 
 interface DeletedRegistrationInformationProps {
   registration: Registration;
+  unpublishingNote: PublicationNote;
 }
 
-export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+function extractId(createdBy: string) {
+  return '/cristin/person/' + createdBy.split('@')[0];
```

Comment:
for å være prepared til dagen hvor endepunktet bytter navn
```suggestion
  return CristinApiPath.Person + createdBy.split('@')[0];
```

Reply:
Ble faktisk litt overrasket over at dette virket, for ikke er det en ID (URI) og ikke er det en identifier (løpenummer). 
Men det er nok fordi vi har satt axios til å ha `REACT_APP_API_HOST` som BASE_URL, sånn at når kallet gjøres her blir det prefikset med base URL til riktig miljø, og bør dermed virke overalt tror jeg 😅